### ISSUE TITLE
Nudging Free blocks/items crashes the plugin on Linux

### ISSUE THREAD

**Issue by alv1n** [XertroV/tm-editor-plus-plus#15]
Both nudging and rotating free blocks/items immediately crashes E++ plugin on Linux. Editing the position/rotation under block properties works fine. Problem started shortly after Map Together was released.
[OpenPlanet Crash Log](https://pastebin.com/AJDfXi0K)

**Comment by XertroV** [comment]
Yeah, linux allocates lower pointers than windows, which is a bit annoying because the length is useful for checking validity. 
Anyway, that breaks some pointer checking stuff I have I think, which throws (instead of failing silently), which crashes the plugin.

The reason it started around the time Map Together was released is because I refactored the way nudging works.

Will probably take a bit to refactor and test.

**Comment by XertroV** [comment]
This should be fixed (as of a while ago). Closing but if anyone has linux issues please comment/reopen.
